src/routes/demographics/survey-schema.js

```diff
@@ -1,7 +1,9 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Is there a PR up for the schema change ? It's nice to have it split out to its own JSON, how easily can we compare it to what's actually loaded on the server side?

I'm presuming that it is https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/main/schemas/pioneer-core/demographic-survey/demographic-survey.1.schema.json, in which case I guess it's not super directly comparable :) Maybe something to consider for the future.

Reply:
I agree. There is not a PR out and this schema is really just for the purposes of formatting (probably the wrong wording here). This part of the code base will probably be refactored when a real frontend engineer takes over this part of the codebase. It's not the most confusing setup ever, but it was slapped together pretty quickly and it'd be better to use an actual json schema approach or whatever.